src/components/Home.js

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
 
-const Home = () => {
+import React from "react";
+import { Button } from "react-bootstrap";
+
+
+const Home = props => {
   return (
-    <div>
-      <h1> Hello This is Home page</h1>
-    </div>
+    <main className="d-flex flex-column justify-content-center ">
```

Comment:
If we are only centering text then we don't need to use flex. We only need to use `text-center`

Reply:
yes, only centring the text.